internal/compiler/program.go

```diff
@@ -1630,6 +1640,50 @@ func (p *Program) SourceFileMayBeEmitted(sourceFile *ast.SourceFile, forceDtsEmi
 	return sourceFileMayBeEmitted(sourceFile, p, forceDtsEmit)
 }
 
+func (p *Program) GetSymlinkCache() *symlinks.KnownSymlinks {
+	// if p.Host().GetSymlinkCache() != nil {
+	// 	return p.Host().GetSymlinkCache()
+	// }
+	if p.knownSymlinks == nil {
+		p.knownSymlinks = symlinks.NewKnownSymlink(p.GetCurrentDirectory(), p.UseCaseSensitiveFileNames())
+		// In declaration-only builds, the symlink cache might not be populated yet
+		// because module resolution was skipped. Populate it now if we have resolutions.
```

Comment:
I don’t recall exactly how this happened in Strada, but I don’t think this comment applies in Corsa.